backend/src/controllers/profileController.ts

```diff
@@ -0,0 +1,79 @@
+import { v4 } from "uuid";
+import { createProfileSchema, getProfileSchema } from "../helpers/schemas";
+import dbPool from "../db/index";
+import { NextFunction, Request, Response } from "express";
+
+interface User {
+  id: string;
+  name: string;
+  email: string;
+  hobby: string;
+}
+
+class ProfileRepository {
```

Comment:
This class must be injected into the service later, therefore must implement interface

Reply:
Also, keep java convention for 1 class per file, so it's easier to change